frontend/src/styles/global.css

```
@@ -49,18 +49,21 @@
    * ═══════════════════════════════════════════════════════════════════════════ */
 
   @font-face {
-    font-family: 'Fraunces';
-    src: url('/fonts/Fraunces-Variable.ttf') format('truetype');
+    font-family: "Fraunces";
```

Stylelint (17.2.0)

[error] Unexpected quotes around "Fraunces" (font-family-name-quotes)

(font-family-name-quotes)